migrations/0001_create_inflight_taskactivations.sql

```diff
@@ -1,5 +1,5 @@
 CREATE TABLE IF NOT EXISTS inflight_taskactivations (
-    id UUID NOT NULL PRIMARY KEY,
+    id TEXT NOT NULL PRIMARY KEY,
```

Comment:
Any reason not to encode the UUID as 16 bytes binary blob? That would make the primary index smaller and comparisons quicker

Reply:
Yes, unfortunately BLOBs have the same problem as UUID: they are interpreted numerically. So the value `97033619637148748591279e7866347` would also be encoded to `Inf` in a BLOB.